paper/paper.md

```diff
@@ -1,351 +1,436 @@
+In addition, we include enhanced support for adaptive time-stepping.
+We continue to expand and maintain a set of examples demonstrating project features 
+  through reproduction of results from literature.
+
+The key motivation behind development of `PySDM` has been to offer the community a set of
+  readily reusable building blocks for development and community dissemination 
+  of extensions to particle-based microphysics models.
+To this end, we strive to maintain strict modularity of the PySDM building blocks, separation of
+  functionality and examples, and extensive unit test coverage in the project.
+A user of the package might select from top-level physics options such as the simulation
+  environment, particle processes, and output attributes without requiring a detailed understanding
+  of the CPU and GPU underlying implementations at the superparticle level.
+The separation of physics information from backend engineering is intended to make the
+  software more approachable for both users and developers who wish to contribute to the
+  scientific progress of particle-based methods for simulating atmospheric clouds.
+
+
+# Summary of new features and examples in v2
+
+## New PySDM Features: API in Brief
+`PySDM` v2 includes support for three major enhancements. For a detailed example of running
+  a SDM simulation, we refer to @Bartman_et_al_2022_JOSS. The following API examples
+  can be added or substituted into the v1 API description to run a zero-dimensional box
+  simulation using the new features.
+
+### Collisional Breakup
+The collisional breakup process represents the splitting of two colliding superdroplets
+  into multiple fragments.
+It can be specified as an individual dynamic, as for coalescence in v1, or as a unified
+  `collision` dynamic, in which the probability of breakup versus coalescence is sampled.
+The additional necessary information can be imported via:
+```python
+from PySDM.dynamics.collisions import Collision
+from PySDM.dynamics.collisions.collision_kernels import Golovin
+from PySDM.dynamics.collisions.coalescence_efficiencies import ConstEc
+from PySDM.dynamics.collisions.breakup_efficiencies import ConstEb
+from PySDM.dynamics.collisions.breakup_fragmentations import ExponFrag
+```
+The rate of superdroplet collisions are specified by a collision kernel as in v1, and the
+  breakup process requires two additional `dynamics` specifications: from `coalescence_efficiencies`
+  (probability of coalescence occuring), `breakup_efficiencies` (probability of breakup occuring
+  if not coalescence), and `breakup_fragmentations` (the number
+  of fragments formed in the case of a breakup event). 
+Specifying a breakup-only event requires only a collision kernel, fragmentation function, 
+  and optional breakup efficiency.
 
 ```python
+from PySDM import Builder
+from PySDM.backends import CPU
+from PySDM.dynamics import Collision 
+from PySDM.environments import Box
 from PySDM.physics import si
```

Comment:
But it might be worth moving these sort of standard imports to a prepended file. What do you think @slayoo?

Reply:
I'd still say that not hiding anything has significant added value. If we are worried about too much space being taken by such imports, we can always just do `import PySDM` and then `builder = PySDM.Builder(...)`, etc

Reply:
Yes, I agree. I don't think it's too much in the end.